flexible/java-11/cloudstorage/README.md

```diff
@@ -0,0 +1,29 @@
+# Cloud Storage sample for Google Managed VMs
```

Comment:
"Managed VMs" as a product as been deprecated and turned down. "App Engine Flex" is better.